src/praisonai/praisonai/sandbox/_compat.py

```
@@ -0,0 +1,175 @@
+            
+        try:
+            if hasattr(self._compute, 'shutdown') and self._instance_id:
+                await self._compute.shutdown(self._instance_id)
+            elif hasattr(self._compute, 'stop'):
+                await self._compute.stop()
+                
+            self._is_running = False
+            self._instance_id = None
+            logger.debug("Stopped compute instance")
+            
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)